lang/python/python-yaml-3/Makefile

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (C) 2020 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-yaml-3
+PKG_VERSION:=3.13
```

Comment:
This version is really old.

Reply:
I know it is old, but it was a requirement for docker-compose. I did not check how big of a change would be to make it work with newer pythom-yaml